data/chamberezigbo.json

```
@@ -1,59 +1,79 @@
 {
   "name": "Chamberlain Ezigbo",
   "type": "personal",
+<<<<<<< HEAD:public/data/chamberezigbo.json
   "bio": "JavaScript Engineer | Front-end Developer",
-
+=======
+  "bio": "JavaScript Engineer | Full-stack Developer",
+>>>>>>> 444abc75fa64b328ee08acf24eca5a99d01d3a77:data/chamberezigbo.json
+  "avatar": "https://github.com/chamberezigbo.png",
   "links": [
     {
-      "name": "My portfiolo site",
-      "url": "https://shiny-starburst-6f2c53.netlify.app",
-      "icon": "FaLink"
+      "name": "My portfolio site",
+      "url": "https://wondrous-churros-f6aea2.netlify.app",
+      "icon": "link"
```

Invalid icon is used. It should be `FaLink`

```suggestion
      "icon": "FaLink"
```